Formula/flif.rb

```diff
@@ -0,0 +1,75 @@
+class Flif < Formula
+  desc "Free Loseless Image Format"
+  homepage "http://flif.info/"
+  head "https://github.com/FLIF-hub/FLIF.git"
+
+  stable do
+    # patch 'install -s', see https://github.com/FLIF-hub/FLIF/issues/369
+    patch :DATA
```

Comment:
@stek29 It's common that upstreams don't test on macOS/BSD versions of `install`.